csharp/ql/src/semmle/code/csharp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -944,17 +944,35 @@ import ReturnNodes
 
 /** A data flow node that represents the output of a call. */
 abstract class OutNode extends Node {
-  /** Gets the underlying call. */
+  /** Gets the underlying call, where this node is a corresponding output of kind `kind`. */
   cached
-  abstract DataFlowCall getCall();
+  abstract DataFlowCall getCall(ReturnKind kind);
 }
 
 private module OutNodes {
+  private import semmle.code.csharp.frameworks.system.Collections
+  private import semmle.code.csharp.frameworks.system.collections.Generic
+
   private DataFlowCall csharpCall(Expr e, ControlFlow::Node cfn) {
     e = any(DispatchCall dc | result = TNonDelegateCall(cfn, dc)).getCall() or
     result = TExplicitDelegateCall(cfn, e)
   }
 
+  /** A valid return type for a method that uses `yield return`. */
+  private class YieldReturnType extends Type {
```

Comment:
I think this class must be duplicated since it wasn't deleted. I guess it can be deleted from `DataFlowDispatch.qll`

Reply:
You are right, I meant to delte the class from DataFlowDispatch.qll. Good spot!